### PR TITLE
Update community page

### PR DESCRIPTION
**Pull request by davydovanton**
Hey,
I start working on new community page (https://github.com/hanami/hanami.github.io/issues/301). Here you can find base PR with a new structure.

<details> 
  <summary> New community page screenshot </summary>
  
</details>

## What the next
I think we need to create some issues with:
- new icons for each link
- update texts (I used texts from crystal page)

/cc @hanami/core

### CODE REVIEW

**Comment by AlfonsoUceda** [comment]
@davydovanton I like it, @cllns could your add your magical touch of your english skills? :P

**Comment by cllns** [comment]
@davydovanton can you take a look at the changes I pushed and let me know what you think? I renamed the section headers to what each one was, rather than the use for them. I think you intentionally had it that way, so let me know if you prefer that.

I'm not sure we should list the forum as the first go-to place for questions. It's not used very much. I think we *want* it to be used more, but Gitter is currently better for questions. Not sure how to handle that :)

**Comment by davydovanton** [comment]
@cllns teamwork 👍 
gif

File: source/community.html.erb [hunk 1]
```diff
@@ -2,12 +2,66 @@
...
+<p>Find and join a local Hanami user group and spread the word about Hanami. Share experiences and help others to get started!</p>
+<ul>
+  <li><a href="http://www.meetup.com/hanamirb-sp-brazil-meetups" target="_blank">Hanami User Group @ São Paulo, Brazil</a> (<a href="https://twitter.com/hanamirb_sp" target="_blank">@hanamirb_sp</a>)</li>
+</ul>
+
+<h2 class="page-header">Contribute to Hanami</h2>
+<p>Help is welcome and much appreciated, whether you are an experienced developer or just looking for sending your first pull request. Please check the open tikets.</p>
+<a href="https://github.com/search?utf8=%E2%9C%93&q=user%3Ahanami+state%3Aopen+label%3Aeasy+label%3Ahelp-wanted&type=Issues&ref=searchresults" target="_blank">Open Tickets</a>
+
+<h2 class="page-header">Images</h2>
```
Comment (jodosha) [inline source/community.html.erb]: Maybe s/Images/Logo ?

File: source/community.html.erb [hunk 2]
```diff
@@ -2,12 +2,66 @@
...
+<p>Hanami is fully open source. That's why you can download our logo in different formats:</p>
```
Comment (jodosha) [inline source/community.html.erb]: I'd rephrase it with:

"Hanami logo it's &copy; Luca Guidi, but you can download and use it for non-commercial scopes."

**Comment by cllns** [comment]
addressed your concerns, @jodosha :)

**Comment by jodosha** [comment]
Quoting @davydovanton 
> update texts (I used texts from crystal page)

We can't copy and paste somebody's else work. I'd like to update the texts **before** to merge/publish this.

**Comment by davydovanton** [comment]
@jodosha good point 👍 
@cllns @hanami/community could you help me with it? 😊

**Comment by oana-sipos** [comment]
@davydovanton Yes, sure, I will have a look tonight.

**Comment by marionschleifer** [comment]
@davydovanton @jodosha I'll have a look at it tomorrow 👍

**Comment by marionschleifer** [comment]
@davydovanton @cllns @jodosha @oana-sipos 

Hey guys, I think this community page is a great idea. Please check my last commit. I rewrote the text that was taken from the crystal community page. I also changed the order a bit. Feel free to give feedback, I can also change it back. 

**Questions**
* Do we want another section where we list talks about Hanami at conferences?
* Do we want to include a picture that represents the community spirit?

**Comment by oana-sipos** [comment]
@marionschleifer the texts look good to me, thanks for taking care of this! I really like the idea of having talks about Hanami somewhere on the website.

**Comment by davydovanton** [comment]
@marionschleifer text is excellent 👏 

About question:
1. Great idea 👍 
2. For me, "cosmetic" images will overload this page. WDYT?

**Comment by marionschleifer** [comment]
@davydovanton @jodosha @oana-sipos @cllns 

Hey guys, I updated the community page with conference talks.

Please have a look and get back to me, if I should change anything 😃

**Comment by davydovanton** [comment]
Awesome. All looks perfect 👍 
@cllns @marionschleifer thanks for you work with text 👏 

teamwork
tumblr_mwu054zuzw1slfdfko1_400

**Comment by jodosha** [comment]
I added the Twitter hashtag:

screen shot 2017-04-12 at 15 43 39

Feel free to revert the commit if you think it isn't a good idea.

**Comment by jodosha** [comment]
@marionschleifer @davydovanton @cllns you did a great job! 👏 💚 🌸 

Shall we link Team and Contributors too?

**Comment by davydovanton** [comment]
Looks great 👏 
I think the contributions link is unnecessary here

File: source/community.html.erb [hunk 3]
```diff
@@ -2,12 +2,75 @@
...
+<a href="https://stackoverflow.com/questions/tagged/hanami" target="_blank">hanami tag</a>
+
+<h2 class="page-header">Awesome Hanami</h2>
+<p>The awesome-hanami Github repository lists Hanami gems, libraries and projects that use Hanami. Also, there are some useful links to know more about Hanami and its users.</p>
+<a href="http://awesome-hanami.org" target="_blank">awesome-hanami.org</a>
+
+<h2 class="page-header">Mailing List</h2>
+<p>To stay updated with the latest releases, to receive code examples, implementation details and announcements, please consider to subscribe to the <a href="/mailing-list">Hanami mailing list</a>.</p>
+
+<h2 class="page-header">Conference Talks about Hanami</h2>
```
Comment (joneslee85) [inline source/community.html.erb]: IMHO this section could be in a separate section. What do you think about a separate section for `Learning`?

**Comment by davydovanton** [comment]
folks, let's merge this great changes?

**Comment by marionschleifer** [comment]
@davydovanton From my side, it can be merged 👍

**Comment by davydovanton** [comment]
thanks for review folks 👍
